### DISCUSSION TITLE
Request/Appreciate

### DISCUSSION THREAD

**Discussion by praveen4030** [graph-cn/flutter_graph_view#2] [General]
Really cool stuff here! Just need more customization to change vertex and edge ui.

Appreciate your effort in putting this package for the community!

**Comment by CorvusYe** [comment]
Thanks for the affirmation.
We're on the same page. Vertex and edge representation and style configuration features are coming soon, probably in a week.

**Comment by CorvusYe** [comment]
> Really cool stuff here! Just need more customization to change vertex and edge ui.
> 
> Appreciate your effort in putting this package for the community!

Hi, @praveen4030 

A new version [0.0.1+7](https://pub.flutter-io.cn/packages/flutter_graph_view) has just been released, which supports the custom UI of vertexes and edges.

You can see if it meets the requirements.
